src/compile/facet.ts

```diff
@@ -127,15 +127,7 @@ export class FacetModel extends Model {
       if (facet[channel]) {
         const axisSpec = facet[channel].axis;
         if (axisSpec !== false) {
-          let vlOnlyAxisProperties: VlOnlyAxisBase = {};
-          VL_ONLY_AXIS_PROPERTIES.forEach(function(property) {
-            if (config.facet.axis[property] !== undefined) {
-              vlOnlyAxisProperties[property] = config.facet.axis[property];
-            }
-          });
-
           const modelAxis = _axis[channel] = {
```

Comment:
We don't need `VL_ONLY_AXIS_PROPERTIES`, but still need to copy `config.facet.axis` as Vega has no facet config.